prerequisites-kubernetes.adoc

```diff
@@ -3,6 +3,8 @@ include::https://raw.githubusercontent.com/redhat-developer-demos/rhd-tutorial-c
 The following CLI tools are required for running the exercises in this tutorial. 
 Please have them installed and configured before you get started with any of the tutorial chapters.
 
+NOTE: You may install Docker instead of Podman.  Podman is preferred because there is no license requirement.
```

Comment:
Maybe I will put in the note, that if you have docker then it's fine and nothing to be done, but if you have any license problems or not docker installed, then use podman

Reply:
Also, maybe I will add a warning saying in the tutorials we are using the `docker` command, if you use podman just replace `docker` with `podman` and done as podman is fully compatible with docker arguments